src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ChunkOutputWidget.java

```diff
@@ -242,6 +243,12 @@ public int getState()
       return state_;
    }
 
+   public void setId(String id)
+   {
+      this.getElement().setPropertyString("id",
```

Comment:
We should use CSS class names rather than IDs, for two reasons:

1. An element can only have one ID, but it can have multiple classes. So once we use the element's ID for this addressability feature, we can't use the ID for anything else.
1. It is considered best practice for a document to have only one element with a given ID, but as chunk names can be duplicated we could also wind up with duplicate IDs. It's normal (and expected) for the same CSS class to be assigned to multiple elements.